cli/stop_file_writer.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import sys
+import time
+from datetime import datetime, timedelta
+from time import time as current_time
+
+from cli.start_file_writer import is_empty
+from file_writer_control import JobHandler, JobState, WorkerCommandChannel
+
+
+def cli_parser() -> argparse.Namespace:
+    """
+    Parser for the command line interface.
+    """
+
+    fw_parser = argparse.ArgumentParser(
+        fromfile_prefix_chars="@", description="FileWriter Stopper"
+    )
+
+    fw_parser.add_argument(
+        "-s",
+        "--stop",
+        metavar="stop",
+        type=str,
+        help="Stop FileWriter immediately.",
+    )
+    fw_parser.add_argument(
+        "-sa",
+        "--stop_after",
+        metavar="stop_after",
+        nargs=2,
+        type=str,
+        help="Stop FileWriter after a given time in seconds.",
+    )
+    fw_parser.add_argument(
+        "-b",
+        "--broker",
+        metavar="kafka_broker",
+        type=str,
+        default="localhost:9092",
+        help="Kafka broker port.",
+    )
+    fw_parser.add_argument(
+        "-t",
+        "--topic",
+        metavar="consume_topic",
+        type=str,
+        required=True,
+        help="Name of the Kafka topic to be consumed.",
```

Comment:
`"Name of the Kafka topic to listen to commands and send status to"`

Reply:
FYI: I also made same changes in your `start_file_writer` script in https://github.com/ess-dmsc/file-writer-control/pull/27

Reply:
Saw it now, thank you.